src/dotnet/commands/dotnet-test/TestRunners/TestRunner.cs

```diff
@@ -51,9 +51,10 @@ private ICommand CreateTestRunnerCommand()
             var commandArgs = _argumentsBuilder.BuildArguments();
 
             return _commandFactory.Create(
-                _testRunner,
+                $"dotnet-{_testRunner}",
                 commandArgs,
-                new NuGetFramework("DNXCore", Version.Parse("5.0")));
+                new NuGetFramework("DNXCore", Version.Parse("5.0")),
+                Constants.DefaultConfiguration);
```

Comment:
We had it at one point, although it was a whole different dotnet-test at that point :smile: 

build scripts currently build the tests with the configuration set, so I'm honestly confused how the driver is finding the appropriate deps file for the test runner right now.
https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/TestTargets.cs#L159